service/src/test/kotlin/app/cash/backfila/service/BackfillRunnerTest.kt

```diff
@@ -839,6 +851,7 @@ class BackfillRunnerTest {
           Result.success(RunBatchResponse.Builder().build())
         )
         waitForOtherCoroutines()
+        delay(2000)
```

Comment:
Maybe add a comment to why?

Reply:
I meant to replace this before merging

Reply:
aha ok that works too

Reply:
I'll have a quick follow to fix this and make tests way better